app/controllers/models_controller.rb

```diff
@@ -6,6 +6,13 @@ class ModelsController < ApplicationController
 
   def index
     @models = resource_base_search_and_page
+    @store = {}
+    Model.all().map do |m|
+      @store[m.id] = {
+        :hosts_count => m.hosts_count,
+        :actions => m.actions_by_user(User.current)
+      }
```

Comment:
this can be written as
```ruby
    @store = {}
    resource_base_search_and_page.each do |m|
      @store[m.id] = {
        :hosts_count => m.hosts_count,
        :actions => m.actions_by_user(User.current)
      }
    end
```